Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx

```diff
@@ -18,6 +22,20 @@ const LoginPage = () => {
                     label="Password"
                     type="password"
                 />
+                <ToggleButtonGroup
```

Comment:
What will be the difference between login in as a student vs an employer? Do we use the same credentials?

Reply:
Not much difference, It'll be for back end because they won't be stored in the same tables